spec/defines/option_spec.rb

```diff
@@ -101,6 +101,19 @@
           )
         }
       end
+
```

Comment:
@welchnut @rehanone could you explain what's going on here?  I'm trying to apply the changes to this pull request and the tests are failing at this point.   I don't understand what line 113 is doing.

Reply:
It's been awhile since I've looked at this so I could be wrong but I think the issue I was running into is that I don't know how to edit the tests to allow a parameter as a hash. Our samba server has been running using this commit for a few months now and I've been meaning to try to fix the tests to get this merged but haven't had the time to do so.

Reply:
I don't know how to write tests at all let alone how to tackle a hash.  My experience with Augeas is about zero, too.